### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Santosh-Konappanavar**
### In this milestone I,

- Added missing JSX elements to the project.
- Final design  matched Zeplin's design.
- Used the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -8,27 +13,218 @@
...
+  height: 3.5rem;
+  background-color: #ffff;
+  margin: 0 2rem;
+}
+
+.header1 {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: #0290ff;
```
Comment (brainconnect93) [inline src/App.css]: - [ ] Hello @Santosh-Konappanavar, I notice repetition of `display: flex;` in all your container, One golden tip in styling is that we can create a class with any styling that is repeating and then give that class to any element we want. for example I can write
```CSS
.flex {
display: flex;
}
```
and then give `flex` class to any container I want. you can apply this rule to save a lot of lines in your CSS.

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @Santosh-Konappanavar, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

- Descriptive Pull Request title and summary. ✔️  
- Styling of the bookstore CMS project ✔️ 
- Zeplin Design template used for the design ✔️ 
- There is good use of Git flow ✔️ 

### Changes Required

- [x] Kindly work on the changes i requested from you via the video demonstration below 👇 

https://user-images.githubusercontent.com/78634841/231871862-773fb882-8df2-49bb-8cfc-25633fc7f9e3.mp4

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Santosh-Konappanavar** [comment]
Hi @brainconnect93 , Thank you so much for reviewing my code. Please click on Books and category name it will show the details of the books name and inputs.

File: src/App.css (hunk 1 above)
Comment (Santosh-Konappanavar) [reply src/App.css]: Hello, **@brainconnect93**  Thanks for your kind suggestion I made changes as mentioned above.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Santosh-Konappanavar 

Your project is complete! Great job for implementing the changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Great design 
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**